raiden/tests/fixtures/variables.py

```diff
@@ -181,7 +182,27 @@ def blockchain_private_keys(blockchain_number_of_nodes, blockchain_key_seed):
 @pytest.fixture(scope='session')
 def port_generator(request):
     """ count generator used to get a unique port number. """
-    return count(request.config.option.initial_port)
+
```

Comment:
yeah I also wanted to say this could have been a different PR, but it's small enough and quite nice :)